lingua_franca/lang/parse_es.py

```diff
@@ -338,12 +339,12 @@ def clean_string(s):
             " ").replace(
             "_",
             "")
-        # handle synonims and equivalents, "tomorrow early = tomorrow morning
-        synonims = {"mañana": ["amanecer", "temprano", "muy temprano"],
+        # handle synonryms and equivalents, "tomorrow early = tomorrow morning
```

Comment:
I seem to have replaced a typo in this comment with a different typo

Reply:
Thanks, updated the commit.